stable/weave-cloud/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+Verify all Pods are running
```

Comment:
Will this work without a service token? If not is it worth doing something like the following?

https://github.com/kubernetes/charts/blob/master/stable/sumokube/templates/NOTES.txt#L1